sympy/polys/polytools.py

```diff
@@ -61,7 +61,18 @@
 
 @public
 class Poly(Expr):
-    """Generic class for representing polynomial expressions. """
+    """Generic class for representing polynomial expressions.
+        
+        Examples
+        ========
+
+        >>> from sympy import Poly
+        >>> from sympy.abc import x
+
+        >>> poly(x*(x**2 + x - 1)**2)
```

Comment:
The examples should use `Poly`, not `poly`, since that's what this is the docstring of.